src/theme/stylus/page.styl

```diff
@@ -2,20 +2,18 @@
 
 .page-wrapper {
     box-sizing: border-box
+    left: 0
+    position: absolute
```

Comment:
Don't really dig having `position: absolute`, but can't think of another way to do it either.